src/Uno.Templates/content/unoapp/global.json

```
@@ -0,0 +1,6 @@
+{
+  "msbuild-sdks": {
+    "Uno.Sdk": "$UnoWinUIVersion$",
```

```suggestion
    "Uno.Sdk": "$UnoVersion$",
```
Should this be `$UnoVersion$` ?

---

No that isn't what the Template uses to update the version